cmd/ipfs-cluster-ctl/graph.go

```diff
@@ -31,8 +31,11 @@ import (
 type nodeType int
 
 const (
-	tCluster nodeType = iota // The cluster node type
-	tIpfs                    // The IPFS node type
+	tSelfCluster    nodeType = iota //  cluster self node
+	tCluster                        //  cluster node
+	tTrustedCluster                 //  trusted cluster node
+	tIPFS                           //  IPFS node linked to a Cluster node
+	tSwarmIPFS                      //  IPFS node linked to a tIPFS node
```

Comment:
What are tSwarmPeers vs tIPFS ? Am I understanding it right that you are printing nodes in the IPFS swarm that are not connected to any cluster peer? Does this mean that if I run connect graph on an IPFS peer connected to 400 other ipfs peers in the network I am going to get 400 boxes? If so, please remove. We're only interested in ipfs peers attached to cluster peers.